### DISCUSSION TITLE
Can add a way to change Drawer container backgound color?

### DISCUSSION THREAD

**Discussion by chuoke** [quasarframework/quasar#15753] [Ideas / Proposals]
For now drawer container background color is always `white`.  And there is no way to change conveniently.
This is not very friendly for implementing a particular style of layout, such as this: https://codepen.io/chuoke/pen/BaqRpWE?editors=101. 

![image](https://user-images.githubusercontent.com/17611457/234481637-8cc6b5f9-636b-4fd4-8e8d-0e6ceb801ded.png)

**Comment by suleimanas** [comment]
css:
```
.q-drawer {
  background: transparent
}
```

https://github.com/quasarframework/quasar/discussions/15904
